litmus-portal/frontend/public/icons/yellowCloudIcon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="74" height="74" viewBox="0 0 74 74" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M66.0508 47.9562C62.4191 47.9562 59.3509 50.4076 58.4045 53.7437H50.7305V45.2715H58.5659C67.0764 45.2715 74 38.3404 74 29.8207C74 22.9899 69.4744 16.9891 63.0497 15.0347C61.1077 7.24987 54.0881 1.66651 45.9583 1.66651C42.8068 1.66651 39.7879 2.48009 37.1236 4.03447C34.3019 1.45122 30.602 0 26.7146 0C18.3761 0 11.5609 6.65434 11.2889 14.9366C4.68946 16.7832 0 22.8322 0 29.8207C0 38.3404 6.92363 45.2715 15.4341 45.2715H23.2695V53.7437H15.5955C14.6493 50.4076 11.5809 47.9562 7.94922 47.9562C3.56602 47.9562 0 51.5261 0 55.914C0 60.302 3.56602 63.8718 7.94922 63.8718C11.5809 63.8718 14.6491 61.4204 15.5955 58.0843H25.4375C26.6348 58.0843 27.6055 57.1126 27.6055 55.914V45.2715H34.832V58.3876C31.4996 59.3349 29.0508 62.4066 29.0508 66.0422C29.0508 70.4301 32.6168 74 37 74C41.3832 74 44.9492 70.4301 44.9492 66.0422C44.9492 62.4066 42.5004 59.335 39.168 58.3876V45.2715H46.3945V55.914C46.3945 57.1126 47.3652 58.0843 48.5625 58.0843H58.4045C59.3507 61.4204 62.4191 63.8718 66.0508 63.8718C70.434 63.8718 74 60.302 74 55.914C74 51.5261 70.434 47.9562 66.0508 47.9562ZM7.94922 59.5312C5.95686 59.5312 4.33594 57.9085 4.33594 55.914C4.33594 53.9195 5.95686 52.2968 7.94922 52.2968C9.94158 52.2968 11.5625 53.9195 11.5625 55.914C11.5625 57.9085 9.94158 59.5312 7.94922 59.5312ZM40.6133 66.0422C40.6133 68.0367 38.9924 69.6594 37 69.6594C35.0076 69.6594 33.3867 68.0367 33.3867 66.0422C33.3867 64.0476 35.0076 62.425 37 62.425C38.9924 62.425 40.6133 64.0476 40.6133 66.0422ZM15.4341 40.9309C9.31461 40.9309 4.33594 35.947 4.33594 29.8207C4.33594 24.3453 8.41273 19.6194 13.8186 18.8279C14.9594 18.661 15.7699 17.6293 15.6636 16.4802C15.6322 16.1412 15.6163 15.7948 15.6163 15.4506C15.6163 9.32469 20.595 4.34078 26.7144 4.34078C29.9794 4.34078 33.0583 5.76161 35.1616 8.23896C35.9 9.10868 37.185 9.25713 38.1021 8.57883C40.3758 6.8964 43.0924 6.00715 45.9582 6.00715C52.5529 6.00715 58.0371 10.8512 59.0623 17.1524C59.2072 18.0434 59.8881 18.7515 60.7719 18.9307C65.9243 19.9749 69.6639 24.5547 69.6639 29.8207C69.6639 35.947 64.6852 40.9309 58.5658 40.9309H15.4341ZM66.0508 59.5312C64.0584 59.5312 62.4375 57.9085 62.4375 55.914C62.4375 53.9195 64.0584 52.2968 66.0508 52.2968C68.0431 52.2968 69.6641 53.9195 69.6641 55.914C69.6641 57.9085 68.0431 59.5312 66.0508 59.5312Z" fill="#F6B92B"/>
```

Comment:
the `fill` parameter of the `path` should be empty and independent of color. The name of the icon should not be color specific. Please correct it.

Reply:
> the `fill` parameter of the `path` should be empty and independent of color. The name of the icon should not be color specific. Please correct it.

can you provide the updated/corrected version of the same? this is from figma. for the name i am changing it.

Reply:
just remove `fill` in the svg file and add a style to the icon with `color:theme.palette.someColor`. The name can be cloudIcon

Reply:
This color is not in the theme